test/nodes/test_extractor_translation.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+
+from haystack.pipelines import TranslationWrapperPipeline, ExtractiveQAPipeline
+from haystack.nodes import DensePassageRetriever, EmbeddingRetriever
+from .test_summarizer import SPLIT_DOCS
+
+# Keeping few (retriever,document_store,reader) combination to reduce test time
+@pytest.mark.integration
+@pytest.mark.elasticsearch
+@pytest.mark.summarizer
+@pytest.mark.parametrize(
+    "retriever,document_store,reader",
+    [("embedding", "memory", "farm"), ("elasticsearch", "elasticsearch", "farm")],
+    indirect=True,
+)
+def test_extractive_qa_pipeline_with_translator(
```

Comment:
This tests the `TranslationWrapperPipeline`.
This test replaces test_summarization_pipeline_with_translator: since the Summarizer no longer changes  the document content, it doesn't work well with the `TranslationWrapperPipeline`